sdk/core/src/options.rs

```diff
@@ -15,7 +15,7 @@ use std::time::Duration;
 ///     .retry(RetryOptions::default().max_retries(10u32))
 ///     .telemetry(TelemetryOptions::default().application_id("my-application"));
 /// ```
-#[derive(Clone, Debug, Default)]
```

Comment:
Why was this removed? I think what there is now is equivalent only more verbose.

Reply:
Ah, can you use [`cfg_attr`](https://stackoverflow.com/questions/42551113/is-it-possible-to-conditionally-enable-an-attribute-like-derive) for this?